_posts/2020-07-24-when-scenarios-and-reality-diverge/when-scenarios-and-reality-diverge.Rmd

```diff
@@ -0,0 +1,380 @@
+  geom_ribbon(aes(ymin = aligned_portfolio, ymax = portfolio), fill = primary_green, alpha = 0.1) + 
+  theme_2dii_ggplot() + 
+  scale_y_continuous(limits = c(0.5,NA)) + 
+  scale_color_manual(values = c(primary_green, primary_blue)) + 
+  labs(x = "Year", y = "t CO2 per t steel", title = "Figure 1: Portfolio alignment concept")
+```
+
+The scaled trajectory of a scenario suggests that a portfolio is consistent with a scenario when the evolution of the indicator (e.g. $CO_2$ intensity) is consistent with what the scenario prescribes. What this entails in practice is from the portfolio's current baseline (e.g. $CO_2$ intensity in 2020), indexing the scenario changes from that baseline over the entire scenario's time horizon. This ensures the differing scopes of the scenario (i.e. macro-economy model) and portfolio (i.e. a specific subset of macro-economic activities) can be bridged by ensuring equal ambition regardless of scope. However, a portfolio's current baseline is not always consistent with the evolution of a scenario — the International Energy Agency's (IEA) [Energy Technology Perspective](https://www.iea.org/topics/energy-technology-perspectives) (ETP) illustrates this point apptly (IEA, 2017). 
+
+ETP 2017 roadmaps a global energy transition that begins in 2014 (IEA, 2017) along three scenario sets, RTS (Reference Technology Scenario), SDS (Sustainable Development Scenario), and B2DS (Beyond 2 Degrees Scenario) (in order of ambition). Each scenario proscribes a set of socioeconomic developments informed by the varying potential of low-carbon technologies to reduce emissions across sectors. These reductions are mapped in terms of the shift from high to low-carbon technologies in units of production or capacity (e.g. TWh, tonnes of steel, etc.).
+
+However, if the current state of the world is any indication, a global, cross-sector energy transition has yet take place (IEA, 2019). To illustrate this point, the IEA assumes that in 2020 the direct $CO_2$ intensity of global steel production has declined by ~20% under ETP's B2DS. In reality, the global $CO_2$ intensity of steel production from 2014 to 2020 has not significantly deviated from $1.8 \ kg \ CO_2 \ per \ kg \ steel$. In effect, The evolution of the scenario and the macro-economy differ, and the longer global emissions continue to rise, the greater the disconnect. And this temporal and evolutionary disconnect between the portfolio (a subset of macro-economic activities) and scenario has implications on the scaled scenario ambition assigned to the portfolio. 
+
+# Data preperation
+
+```{r, warning=FALSE, message=FALSE}
+index_scenario_to_market <- function(
+  data,
+  aviation_intensity = 87.9,
+  steel_intensity = 1.8, 
+  cement_intensity = 0.6
+) {
+  year_seq <- unique(data$year)
+  index_data <- tibble::tibble()
+  for (i in 2:length(year_seq)) {
+    index_data <- data %>% 
+      dplyr::select(
```

Comment:
If you make the call to `library(tidyverse)` visible, then you can get rid of the namespace and, for example, instead of `dplyr::select()` use `select()`. I believe dplyr functions are so popular that removing the namespace will not impair the reader; instead it will help them see more clearly the interesting patterns.